### DISCUSSION TITLE
Spamming

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#1759] [General]
I wanted to ask why KikiManjaro is spamming in the actions tab?

**Comment by RPYgamers** [comment]
My guess is ...
He is kylian Meulin from discord that want to contribute by add color on touch screen control 
Even chat with dev on development channel 

So ..
No spamming
( You also don't spam on GitHub )

**Reply by ghost** [reply]
ok understood so will the same workflow keep popping up as it gets updated until approved or something?

**Comment by Swizzlefowl** [comment]
Yes.
